src/lib.rs

```diff
@@ -439,11 +446,12 @@ impl Compiler {
 
     /// Parses and typechecks `expr_str` followed by extracting metadata from the created
     /// expression
-    pub fn extract_metadata(&mut self,
-                            vm: &Thread,
-                            file: &str,
-                            expr_str: &str)
-                            -> Result<(ast::SpannedExpr<ast::TcIdent<Symbol>>, TcType, Metadata)> {
+    pub fn extract_metadata
+        (&mut self,
+         vm: &Thread,
+         file: &str,
+         expr_str: &str)
+         -> Result<(ast::SpannedExpr<ast::TcIdent<Symbol>>, TcType, Metadata)> {
```

Comment:
Are you using a new version of rustfmt or a different config?

I don't have much of an opinion when it comes how code should be formatted as long as it is consistent (so if anything I just stick to the default since that is simpler).